awx/ui/src/screens/Setting/shared/data.allSettingOptions.json

```diff
@@ -639,6 +639,15 @@
         "unit": "seconds",
         "default": 14400
       },
+      "UI_NEXT": {
+        "type": "boolean",
+        "required": false,
+        "label": "Enable Next Generation User Interface",
```

Comment:
should be "preview of new" here and a few other spots

Reply:
updated

Reply:
fixed